Assets/Scripts/ManagerScripts/RespawnManager.cs

```diff
@@ -52,13 +52,19 @@ public IEnumerator RespawnPlayer(float respawnClipLength)
         AdjustMusicTime();
```

Comment:
Github doesn't let me select lines that you didn't change 😠 But what if we move line 49 in this file (RespawnManager.cs) (the line of code that resets all the fish) to in between lines 59 and 60. I think this will solve the problem where the game freezes when you die (because the respawning of fish treats will happen after the player respawns)

Reply:
Made small edit to my comment (lines 58 and 59) instead of (lines 59 and 60) (essentially right before the countdown